fyrox-ui/src/inspector/mod.rs

```diff
@@ -1025,6 +1021,44 @@ impl Control for Inspector {
             }
         }
 
+        if !message.handled() {
```

Comment:
Right, the point is so that Inspector can know which widget was the destination for the MouseDown message and than use that to set the target for the Menu. If the default widget handling for WidgetMessage::MouseDown handles the MouseDown message, then the original destination of the MouseDown message is forgotten.

There was no documentation to explain what `target` is supposed to be used for. I took it to represent the widget being clicked on, since the handler for Copy Value as String already has a loop to search up through parents to find the actual editor widget. We could move that loop into the MouseDown handler so that the target always points to the editor widget, if you prefer that.